force-app/main/default/classes/ServiceDeliveryController_TEST.cls

```diff
@@ -77,4 +77,73 @@ public with sharing class ServiceDeliveryController_TEST {
             contactId
         );
     }
+
+    @IsTest
+    private static void testGetServicesByProgramEngagementIds() {
+        Id programEngagementId = TestUtil.mockId(ProgramEngagement__c.SObjectType);
+
+        List<Object> expected = new List<Object>();
+
+        serviceStub.withReturnValue(
+            'getServicesByProgramEngagementId',
+            Id.class,
+            expected
+        );
+
+        Test.startTest();
+        ServiceDeliveryController.service = (ServiceService) serviceStub.createMock();
+
+        final List<Object> actual = ServiceDeliveryController.getServicesByProgramEngagementId(
+            programEngagementId
+        );
+
+        Test.stopTest();
+
+        System.assertEquals(
+            expected.size(),
+            actual.size(),
+            'Expected the size of both expected and actual are the same'
```

Comment:
What do you think of... Expected the controller to return the list returned by the service?

Reply:
Yeah.. I can make that change